Bottom: 6f6b396308a72da03ca1963d51667ee6db450cc1
Top:    e6949d1f37d3f0c994103e95d7355b62fd4838c5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:31:38 +0000

Temporarily disable rubber-banding of lines

Lines are being moved endpoint by endpoint, and for some reason this
is causing a greater incidence of tripping up the polygon code under
development.


---

diff --git a/src/move.c b/src/move.c
index 6282e9e..9a4ce64 100644
--- a/src/move.c
+++ b/src/move.c
@@ -926,7 +926,8 @@ MoveObjectAndRubberband (int Type, void *Ptr1, void *Ptr2, void *Ptr3,
     if (!found_line_at_1_end)
       max_multiplier = MAX (max_multiplier, 1.0);
 
-  if (Type == LINE_TYPE)
+//  if (Type == LINE_TYPE)
+    if (0)
     {
       Coord ldx, ldy;
       ldx = (moving_line->Point2.X - moving_line->Point1.X);
